blinkit-app/src/homePage/molecules/logoLocation/LogoLocation.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from "react";
+import InsertSelect from "../../atoms/insertSelect/InsertSelect";
+import "./logoLocation.css";
+import { locations } from "../../../DATA/LOCATION_DATA";
```

Comment:
LogoLocation doesn't convey clearly what the component, LeftHeaderSection would be a better name


This is only being used by LogoLocation, so instead of keeping it anywhere outside, it increases the risk of data pollution.
Create a separate constant file logoLocation.constants or create a folder constants, a file inside the constants folder named 'locations.js' and then import it here.